### ISSUE TITLE
Missing class while generating managers with Maven plugin

### ISSUE THREAD

**Issue by cesdperez** [cmeza20/spring-data-generator#20]
When I run

```
mvn spring-data-generator:managers
```

I get 

```
A required class was missing while executing com.cmeza:spring-data-generator:1.1.7:managers: javafx/util/Pair
```

Full output:
```
/usr/lib/jvm/default-runtime/bin/java -Dmaven.multiModuleProjectDirectory=/opt/projects/gestion-v2/database -Dmaven.home=/opt/intellij-idea-ultimate-edition/plugins/maven/lib/maven3 -Dclassworlds.conf=/opt/intellij-idea-ultimate-edition/plugins/maven/lib/maven3/bin/m2.conf -javaagent:/opt/intellij-idea-ultimate-edition/lib/idea_rt.jar=33075:/opt/intellij-idea-ultimate-edition/bin -Dfile.encoding=UTF-8 -classpath /opt/intellij-idea-ultimate-edition/plugins/maven/lib/maven3/boot/plexus-classworlds-2.5.2.jar org.codehaus.classworlds.Launcher -Didea.version=2018.1.6 com.cmeza:spring-data-generator:1.1.7:managers -f pom.xml
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building gestion-database 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] >>> spring-data-generator:1.1.7:managers (default-cli) > compile @ database >>>
[INFO] 
[INFO] --- maven-resources-plugin:3.0.2:resources (default-resources) @ database ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ database ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] <<< spring-data-generator:1.1.7:managers (default-cli) < compile @ database <<<
[INFO] 
[INFO] --- spring-data-generator:1.1.7:managers (default-cli) @ database ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.686 s
[INFO] Finished at: 2018-08-12T19:52:24-03:00
[INFO] Final Memory: 23M/212M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.cmeza:spring-data-generator:1.1.7:managers (default-cli) on project database: Execution default-cli of goal com.cmeza:spring-data-generator:1.1.7:managers failed: A required class was missing while executing com.cmeza:spring-data-generator:1.1.7:managers: javafx/util/Pair
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>com.cmeza:spring-data-generator:1.1.7
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/checho/.m2/repository/com/cmeza/spring-data-generator/1.1.7/spring-data-generator-1.1.7.jar
[ERROR] urls[1] = file:/home/checho/.m2/repository/org/springframework/boot/spring-boot-autoconfigure/1.5.2.RELEASE/spring-boot-autoconfigure-1.5.2.RELEASE.jar
[ERROR] urls[2] = file:/home/checho/.m2/repository/org/springframework/boot/spring-boot/1.5.2.RELEASE/spring-boot-1.5.2.RELEASE.jar
[ERROR] urls[3] = file:/home/checho/.m2/repository/org/springframework/spring-core/4.3.7.RELEASE/spring-core-4.3.7.RELEASE.jar
[ERROR] urls[4] = file:/home/checho/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
[ERROR] urls[5] = file:/home/checho/.m2/repository/org/springframework/spring-context/4.3.7.RELEASE/spring-context-4.3.7.RELEASE.jar
[ERROR] urls[6] = file:/home/checho/.m2/repository/org/springframework/spring-aop/4.3.7.RELEASE/spring-aop-4.3.7.RELEASE.jar
[ERROR] urls[7] = file:/home/checho/.m2/repository/org/springframework/spring-beans/4.3.7.RELEASE/spring-beans-4.3.7.RELEASE.jar
[ERROR] urls[8] = file:/home/checho/.m2/repository/org/springframework/spring-expression/4.3.7.RELEASE/spring-expression-4.3.7.RELEASE.jar
[ERROR] urls[9] = file:/home/checho/.m2/repository/javax/persistence/persistence-api/1.0.2/persistence-api-1.0.2.jar
[ERROR] urls[10] = file:/home/checho/.m2/repository/org/apache/maven/maven-builder-support/3.3.1/maven-builder-support-3.3.1.jar
[ERROR] urls[11] = file:/home/checho/.m2/repository/com/google/guava/guava/18.0/guava-18.0.jar
[ERROR] urls[12] = file:/home/checho/.m2/repository/org/eclipse/aether/aether-util/1.0.2.v20150114/aether-util-1.0.2.v20150114.jar
[ERROR] urls[13] = file:/home/checho/.m2/repository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar
[ERROR] urls[14] = file:/home/checho/.m2/repository/javax/annotation/jsr250-api/1.0/jsr250-api-1.0.jar
[ERROR] urls[15] = file:/home/checho/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0.3.0/org.eclipse.sisu.inject-0.3.0.jar
[ERROR] urls[16] = file:/home/checho/.m2/repository/org/sonatype/sisu/sisu-guice/3.2.5/sisu-guice-3.2.5-no_aop.jar
[ERROR] urls[17] = file:/home/checho/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar
[ERROR] urls[18] = file:/home/checho/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.21/plexus-interpolation-1.21.jar
[ERROR] urls[19] = file:/home/checho/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.20/plexus-utils-3.0.20.jar
[ERROR] urls[20] = file:/home/checho/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
[ERROR] urls[21] = file:/home/checho/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
[ERROR] urls[22] = file:/home/checho/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
[ERROR] urls[23] = file:/home/checho/.m2/repository/de/vandermeer/asciitable/0.3.2/asciitable-0.3.2.jar
[ERROR] urls[24] = file:/home/checho/.m2/repository/de/vandermeer/ascii-utf-themes/0.0.1/ascii-utf-themes-0.0.1.jar
[ERROR] urls[25] = file:/home/checho/.m2/repository/de/vandermeer/skb-interfaces/0.0.1/skb-interfaces-0.0.1.jar
[ERROR] urls[26] = file:/home/checho/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar
[ERROR] urls[27] = file:/home/checho/.m2/repository/org/antlr/ST4/4.0.8/ST4-4.0.8.jar
[ERROR] urls[28] = file:/home/checho/.m2/repository/org/antlr/antlr-runtime/3.5.2/antlr-runtime-3.5.2.jar
[ERROR] urls[29] = file:/home/checho/.m2/repository/org/antlr/antlr4/4.5.1/antlr4-4.5.1.jar
[ERROR] urls[30] = file:/home/checho/.m2/repository/de/vandermeer/char-translation/0.0.2/char-translation-0.0.2.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR] 
[ERROR] -----------------------------------------------------: javafx.util.Pair
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException

Process finished with exit code 1
```

**Comment by cesdperez** [comment]
looks like it's an issue with OpenJDK https://gist.github.com/androidfred/bc64da9e6a355b984d37439ed63ae16b

I can send a PR for replacing `javafx.util.Pair` with something else if you don't mind.

**Comment by cmeza20** [comment]
Thank you @cesdperez 

Your fix was published in version 1.1.8

Best regards
